desktop/src/main/kotlin/io/spine/examples/pingh/desktop/Tray.kt

```diff
@@ -163,6 +260,40 @@ private fun mouseEventHandler(state: AppState, menu: Menu) =
         }
     }
 
+/**
+ * Default data for styling the tray icon.
+ *
+ * Dimensions and coordinates are specified in pixels and adjusted for screen [density].
+ */
+@Suppress("MagicNumber" /* Colors are defined using RGB components. */)
+private class TrayStyle(private val density: Density) {
+    val boxSize = AwtSize(22.adjusted, 22.adjusted)
+    val iconSize = AwtSize(16.adjusted, 16.adjusted)
+    val iconPosition = Point(3.adjusted, 3.adjusted)
+    val badgeSize = AwtSize(17.adjusted, 12.adjusted)
+    val badgePosition = Point(3.adjusted, 10.adjusted)
+    val badgeColor = Color(240, 77, 63)
+    val fontSize = 8.adjusted
+    val fontName = "San Francisco"
+    val fontColor = Color.WHITE!!
+
+    /**
+     * Adapts standard pixel value to fit the screen density.
+     */
+    private val Int.adjusted: Int
+        get() = round(this * density.density).toInt()
+}
+
+/**
+ * Holds a 2D integer size.
```

Comment:
The type with this name should have some explanation of the purpose.

Right now we are not here nor there. Please rephrase.